service/dap/server.go

```diff
@@ -1684,10 +1757,10 @@ func (s *Server) doCommand(command string) {
 
 		// TODO(polina): according to the spec, the extra 'text' is supposed to show up in the UI (e.g. on hover),
 		// but so far I am unable to get this to work in vscode - see https://github.com/microsoft/vscode/issues/104475.
-		// Options to explore:
-		//   - supporting ExceptionInfo request
-		//   - virtual variable scope for Exception that shows the message (details here: https://github.com/microsoft/vscode/issues/3101)
 		// In the meantime, provide the extra details by outputing an error message.
+		// TODO(suzmue): the ExceptionInfo request is now supported and displays the error message, however if there
+		// is no threadId set in the stopped event, then the ExceptionInfo response is not visible. We should be able
+		// to delete this output event, once we solve that issue.
```

Comment:
I remember that this happens, but I can't remember when. Could you remind me please?

Reply:
Oh that actually wouldn't be an error, that would be stopped at a breakpoint. Not sure if there is a case where this comment applies actually.

Reply:
You never know. I always find new surprises :) You could check if goroutine is set and if not then send an output event.